src/main.js

```diff
@@ -768,6 +768,10 @@ function isTrustedURL(url) {
   if (typeof url === 'string') {
     parsedUrl = new URL(url);
   }
+  if (!parsedUrl) {
```

Comment:
Looking at this again, this check should probably be the first line of this function.

Reply:
Yes, agreed. No need to run the rest is there isn't anything to check. :)

Reply:
Also, `new URL(url)` throws an error if the provided URL isn't valid, so it should probably be wrapped in a try...catch, no?

Reply:
With the new check, URL's shouldn't be invalid, but might be better safe than sorry?